### PR TITLE
Install details and Uninstall

### PR DESCRIPTION
**Pull request by slashtechno**
Since /usr/local/bin is modified it means that Pi-Apps does modify files/folders outside of the home directory. I put this in the README as well as a way to uninstall. For some reason I can not inline pi-apps in line 26.

**Also I believe the README should be shortened**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,10 +22,11 @@ wget -qO- https://raw.githubusercontent.com/Botspot/pi-apps/master/install | bas
 git clone https://github.com/Botspot/pi-apps
 ~/pi-apps/install
 ```
-
 </details>
+The install script ensures YAD is installed, creates two menu buttons, and an autostarted updater. Nothing is modified outside your home directory execpt one file to allow `pi-apps` in a terminal.
+
+To uninstall Pi-Apps run `sudo rm //usr/local/bin/pi-apps && rm -r //home/pi/pi-apps`
```
Comment (Itai-Nelken) [inline README.md:28]: pi-apps has a uninstall script. `~/pi-apps/uninstall`.

File: README.md [hunk 2]
```diff
@@ -22,10 +22,11 @@ wget -qO- https://raw.githubusercontent.com/Botspot/pi-apps/master/install | bas
 git clone https://github.com/Botspot/pi-apps
 ~/pi-apps/install
 ```
-
 </details>
+The install script ensures YAD is installed, creates two menu buttons, and an autostarted updater. Nothing is modified outside your home directory execpt one file to allow `pi-apps` in a terminal.
```
Comment (Itai-Nelken) [inline README.md:26]:
```suggestion
The install script ensures YAD is installed, creates two menu buttons, an autostarted updater, and a file (`/usr/local/bin/pi-apps`) that allows pi-apps to be run from terminal. Nothing is modified outside your home directory.
```

**Review by Itai-Nelken — CHANGES_REQUESTED** [review]
I changed the part about /usr/local/bin/pi-apps:
as I understand it, the 'no files are modified outside of your home directory' part means no apps modify files outside of your home directory other then things like menu buttons and executables that go in /usr/bin and /usr/local/bin etc.

Pi Apps also has a uninstall script.

**Comment by Itai-Nelken** [comment]
also I think the Readme isn't to long, it explains everything you need to know about pi-apps, maybe something better would be to add all the 'how it works' part in a different file or a dropdown details html thing like with the 'manual installation'.
but I still think its unnecessary to do any of those things.

File: README.md (hunk 2 above)
Comment (slashtechno) [reply README.md:26]: I agree,

File: README.md (hunk 1 above)
Comment (slashtechno) [reply README.md:28]: Oh okay.

**Comment by slashtechno** [comment]
@Itai-Nelken I agree with you. I think that Pi-Apps needs a Github Pages site.

**Comment by Botspot** [comment]
> Pi Apps also has a uninstall script.

Speaking of that, I just improved it so that it would remove the terminal command, and to notify the user that `~/pi-apps` still exists.

**Pull request merged**